tests/pipelines/test_pipelines_common.py

```diff
@@ -673,7 +672,9 @@ def _test_attention_slicing_forward_pass(
         output_with_slicing = pipe(**inputs)[0]
 
         if test_max_difference:
-            max_diff = np.abs(to_np(output_with_slicing) - to_np(output_without_slicing)).max()
```

Comment:
https://github.com/huggingface/diffusers/pull/4812/files#r1307509747 would be better